docs-website/sphinx/apidocs/sdk-v2/entities.rst

```diff
@@ -0,0 +1,44 @@
+Entities
+=======
+
+The DataHub SDK provides a set of entities that can be used to interact with DataHub's metadata.
+
+Dataset
+-------
+
+.. automodule:: datahub.sdk.dataset
```

Comment:
<img width="728" alt="image" src="https://github.com/user-attachments/assets/ed457f92-a40b-4d93-a1e6-1fa8f06ef4fb" />

that `\` shouldn't be there

Reply:
added parsing for this